packages/components/addon/components/hds/app-footer/index.hbs

```diff
@@ -0,0 +1,23 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class={{this.classNames}} ...attributes>
+  <ul class="hds-app-footer__list">
+    {{yield to="linksBefore"}}
+
+    <Hds::AppFooter::Link @href="https://www.hashicorp.com/support">Support</Hds::AppFooter::Link>
+    <Hds::AppFooter::Link @href="https://portal.cloud.hashicorp.com/terms-of-service">Terms</Hds::AppFooter::Link>
+    <Hds::AppFooter::Link @href="https://www.hashicorp.com/privacy">Privacy</Hds::AppFooter::Link>
+    <Hds::AppFooter::Link @href="https://www.hashicorp.com/security">Security</Hds::AppFooter::Link>
+    <Hds::AppFooter::Link @href="mailto:accessibility@hashicorp.com">Accessibility Support</Hds::AppFooter::Link>
+
+    {{yield to="linksAfter"}}
+
+    <Hds::AppFooter::Item class="hds-app-footer__item--copyright"><FlightIcon @name="hashicorp" />
+      © 2023 HashiCorp</Hds::AppFooter::Item>
+  </ul>
+</div>
+
+{{! NOTE: DOM, nested component choices, class names, & styling are still in flux}}
```

Comment:
Note: yield "placeholders" are not yet finalized. May change or move depending on more finalized visual designs.

Reply:
I pushed up a refactored structure for these but it still may change according to final designs.